I18N.md

```diff
@@ -1,22 +1,28 @@
 # Internationalization
 
-App is translated to many different languages through lokalise.com platform.
+App is translated to many different languages through lokalise.com platform. For this purpose, we are using Qt's [TS file format](https://doc.qt.io/qt-6/linguist-ts-file-format.html). 
+There are excellent manuals detailing the translation process for [developers](https://doc.qt.io/qt-6/linguist-programmers.html), [release managers](https://doc.qt.io/qt-6/linguist-manager.html) and [translators](https://doc.qt.io/qt-6/linguist-translators.html) in the Linguist manual (Qt's translation tool).
```

Comment:
Pointing to Qt Linguist might be confusing given the Lokalise workflow, where translators never use Qt Linguist.

Reply:
Yeah, was more about the technicalities how the whole process around TS files works and the related CLI tools; I'll reword it a bit